feature-libs/organization/administration/components/shared/active-guard.directive.ts

```diff
@@ -0,0 +1,55 @@
+import { Directive, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { GlobalMessageType } from '@spartacus/core';
+import { tap } from 'rxjs/operators';
+import { OrganizationItemService } from './organization-item.service';
+import { MessageService } from './organization-message/services/message.service';
+import { BaseItem } from './organization.model';
+
+@Directive({
+  selector: '[cxActiveGuard]',
```

Comment:
Components' and directives's selectors are global. The `cxActiveGuard` is quite general name, but used for very specific organization-related stuff.

Would you mind prefixing all new directives in this PR with `cxOrg...`? i.e. `cxOrgActiveGuard`?

Note: Due to similar reasons, we plan very soon in #9484 to prefix all organization components with `'cx-org-'`.

https://angular.io/guide/styleguide#directive-custom-prefix